src/detect-byte-extract.c

```diff
@@ -38,27 +34,24 @@
 #include "detect-isdataat.h"
 #include "detect-engine-build.h"
 
-#include "app-layer-protos.h"
+#include "rust.h"
 
-#include "flow.h"
-#include "flow-var.h"
-#include "flow-util.h"
+#include "app-layer-protos.h"
 
 #include "util-byte.h"
 #include "util-debug.h"
 #include "util-unittest.h"
 #include "util-unittest-helper.h"
-#include "util-spm.h"
 
 /* the default value of endianness to be used, if none's specified */
-#define DETECT_BYTE_EXTRACT_ENDIAN_DEFAULT DETECT_BYTE_EXTRACT_ENDIAN_BIG
+#define DETECT_BYTE_EXTRACT_ENDIAN_DEFAULT (uint8_t) BigEndian
```

Comment:
`BigEndian` is coming from the Rust enum right?

Reply:
Correct -- `rust/src/detect/lib.rs`